spec/unit/local_system_spec.rb

```diff
@@ -102,33 +102,23 @@
       }.not_to raise_error
     end
 
-    it "raises Machinery::Errors::IncompatibleHost on hosts that can not run machinery" do
+    it "puts an IncompatibleHost message on hosts that can not run machinery" do
```

Comment:
I would prefer something like "shows a warning on hosts that can not run machinery"